test/regression/5076-empty-text-should-not-change-prompt-history/script

```diff
@@ -0,0 +1,6 @@
+ui_out -ignore 4
+ui_in '{ "jsonrpc": "2.0", "method": "keys", "params": [ "xsa<ret>" ] }'
+sleep 0.1
```

Comment:
Using `ui_out -ignore <n>` would be better here to avoid timing based flakyness.